src/run/test/report.ts

```diff
@@ -468,6 +486,41 @@ t.test('not full coverage', async t => {
   process.exitCode = 0
 })
 
+t.test('coverage meets minimum thresholds', async t => {
+  summary = summary95
+  t.testdir({
+    '.tap': {
+      coverage: { 'file.json': '{}' },
+    },
+  })
+  const comments = t.capture(mockTap, 'comment')
+  let openerRan = false
+  const htmlReport = resolve(projectRoot, '.tap/report/index.html')
+  const { report } = await t.mockImport<
+    typeof import('../dist/esm/report.js')
+  >('../dist/esm/report.js', {
+    c8: { Report: MockReport },
+    '@tapjs/core': mockCore,
+    opener: (file: string) => {
+      t.equal(file, htmlReport)
+      openerRan = true
+    },
+  })
+  const config = new MockConfig([])
+  config.statements = 90
+  config.branches = 90
+  config.functions = 90
+  config.lines = 90
+  const logs = t.capture(console, 'log')
+  await report(['html'], config as unknown as LoadedConfig)
+  t.strictSame(logs.args(), [])
+  t.strictSame(comments.args(), [])
+  t.equal(openerRan, true)
+  t.equal(readFileSync(htmlReport, 'utf8'), 'report')
+  t.equal(process.exitCode, 0)
+  process.exitCode = 0
```

Comment:
This should not set the exitCode if the test is failing, because in that case, we don't want to _actually_ set it back to zero. (I can fix this up, no action needed.)